### ISSUE TITLE
`unattended-upgrades` / systemd problem

### ISSUE THREAD

**Issue by eins78** [leihs/leihs#732]
https://madek.slack.com/archives/C80TT63DM/p1562787847004400

**Comment by DrTom** [comment]
```
systemctl stop postgresql
# postgres aus, alle leihs-services auch (weil dependent?)
# ... update passiert, ok
systemctl start postgresql
# postgres startet neu - leihs services nicht!!!
```

**Comment by DrTom** [comment]
somewhat related: 

https://github.com/leihs/leihs/issues/734

**Comment by DrTom** [comment]
Fix underway, pure systemd,  not like Madek (that is more or less broken, too), fix will include #734

**Comment by DrTom** [comment]
fixed as of https://github.com/leihs/leihs/commit/9f78b6b62e7a14f5fbb4f2348efef34bf502a881
